.github/workflows/production_release.yml

```diff
@@ -74,7 +74,7 @@ jobs:
           role-to-assume: arn:aws:iam::046399108653:role/pcluster-manager-github-prod-ProductionDeploy-1I1HYDA5DH7T3
 
       - name: Build and upload Docker image
-        run: ./scripts/build_and_release_image.sh --release
```

Comment:
question: since we are not passing the revision, how are we going to manage the revision arg?

Reply:
This is going to be worked as a new task in our board